koku/koku/configurator.py

```diff
@@ -380,6 +410,36 @@ def get_kafka_topic(requestedName: str):
         """Obtain kafka topic."""
         return KafkaTopics.get(requestedName).name
 
+    @staticmethod
+    def get_kafka_username():
+        """Obtain kafka username"""
+        return LoadedConfig.kafka.brokers[0].sasl.username
+
+    @staticmethod
+    def get_kafka_password():
+        """Obtain kafka password"""
+        return LoadedConfig.kafka.brokers[0].sasl.password
+
+    @staticmethod
+    def get_kafka_sasl_mechanism():
+        """Obtain kafka sasl mechanism"""
+        return LoadedConfig.kafka.brokers[0].sasl.saslMechanism
+
+    @staticmethod
+    def get_kafka_security_protocol():
+        """Obtain kafka security protocol"""
+        return LoadedConfig.kafka.brokers[0].sasl.securityProtocol
```

Comment:
The issue that we are hitting right now in ephemeral is that `sasl` is not configured there. I think instead of these 4 methods, we could just use 1 that returns the sasl config that can be used directly in the Kafka configs.

Maybe something like:
```
    @staticmethod
    def get_kafka_sasl():
        """Obtain kafka sasl config"""
        conf = {}
        sasl = LoadedConfig.kafka.brokers[0].sasl
        if sasl:
            conf["security_protocol"] = sasl.securityProtocol
            conf["sasl_mechanism"] = sasl.saslMechanism
            conf["sasl_plain_username"] = sasl.username
            conf["sasl_plain_password"] = sasl.password
        return conf
```
or something similar.